system_tests/system_tests_sync/test_external_accounts.py

```diff
@@ -100,6 +102,23 @@ def service_account_info(service_account_file):
         yield json.load(f)
 
 
+@pytest.fixture
```

Comment:
We should explain how the AWS tests work somewhere. Something like:

```
# AWS provider tests for AWS credentials
# The test suite will also run tests for AWS credentials. This works as
# follows. (Note prequisite setup is needed. This is documented in
# setup_external_accounts.sh).
# - iamcredentials:generateIdToken is used to generate a Google ID token using
#   the service account access token. The service account client_id is used as
#   audience.
# - AWS STS AssumeRoleWithWebIdentity API is used to exchange this token for
#   temporary AWS security credentials for a specified AWS ARN role.
# - AWS_REGION, AWS_ACCESS_KEY_ID, AWS_SECRET_ACCESS_KEY and AWS_SESSION_TOKEN
#   environment variables are set using these credentials before the test is
#   run simulating an AWS VM.
# - The test can now be run.
```

Reply:
Done. I put this as a comment on the test itself instead of the main fixture it uses